examples/hmc-agent/app/Containerfile-agent

```diff
@@ -0,0 +1,31 @@
+ARG BASE_UBI_IMAGE_TAG=9.5-1742914212
+FROM registry.access.redhat.com/ubi9/ubi-minimal:${BASE_UBI_IMAGE_TAG}
+
+ARG PYTHON_VERSION=3.12
+
+ENV VIRTUAL_ENV=/opt/venv
+ENV PATH=${VIRTUAL_ENV}/bin:$PATH
+
+RUN microdnf install -y dnf openssl-devel && dnf install -y \
+    openblas-devel python${PYTHON_VERSION}-devel python${PYTHON_VERSION}-pip \
+    && dnf clean all \
+    && python${PYTHON_VERSION} -m venv ${VIRTUAL_ENV} \
+    && python -m pip install -U pip uv \
+    && uv pip install wheel build setuptools 'cmake<4' cython
+
+RUN dnf groupinstall -y "Development Tools" && \
+    dnf install -y nodejs && \
+    dnf clean all && \
+    curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y && \
+    export PATH="$HOME/.cargo/bin:$PATH" && \
+    rustup install nightly && \
+    rustup override set nightly
```

Comment:
I dont think this is requied right. Please install what is necessary only.
Same applies to server containerfile too.

Reply:
rust is one of the dependencies for `tiktoken` which is a dependency fro pyyaml. So, without this dependencies for HMC agent can't be installed.

Reply:
ok